src/Agent.Worker/ContainerOperationProvider.cs

```diff
@@ -17,8 +17,11 @@ namespace Microsoft.VisualStudio.Services.Agent.Worker
     [ServiceLocator(Default = typeof(ContainerOperationProvider))]
     public interface IContainerOperationProvider : IAgentService
     {
-        Task StartContainerAsync(IExecutionContext executionContext, object data);
         Task StopContainerAsync(IExecutionContext executionContext, object data);
+        Task CreateContainerNetworkAsync(IExecutionContext executionContext, object data);
+        Task RemoveContainerNetworkAsync(IExecutionContext executionContext, object data);
+        Task StartMultipleContainersAsync(IExecutionContext executionContext, object data);
+        Task StopMultipleContainersAsync(IExecutionContext executionContext, object data);
```

Comment:
how many interface methods do we really need?

Reply:
Not that many :) Cleaned up